testctrl/proto/scheduling/v1/scheduling_service.proto

```diff
@@ -90,6 +90,11 @@ message Component {
 
   // The kind which assigns the responsibilities of this container.
   Kind kind = 2;
+
+  // The Kubernetes node pool where the component should be scheduled. This
+  // does not need to match the node pool's official name, but it must match
```

Comment:
I might just say "This must match the value of the "pool" label..." without talking about official names. I would also say "... to be scheduled on that node." instead of "a chance at scheduling."

Reply:
Thanks, Paulo. Fixed it. :)